go.graphics.swing/src/go/graphics/swing/event/swingInterpreter/GOSwingEventConverter.java

```diff
@@ -253,7 +253,7 @@ private String getKeyName(KeyEvent e) {
 				text = "BACK_SPACE";
 				break;
 			default:
-				text = "";
+				text = "" + e.getKeyChar();
```

Comment:
Yes, we should find a way that handles all of this.

We also need mouse Shift + Alt + ... for the move to commands.

I am just working on Android support for move-to and move-to-and-work